test/models/patient_notification_eligibilty_test.rb

```diff
@@ -0,0 +1,400 @@
+    [
+      { isolation: true },
+      { continuous_exposure: true },
+      { last_date_of_exposure: nil, created_at: 5.days.ago },
+      { last_date_of_exposure: 5.days.ago, created_at: 5.days.ago },
+      { last_date_of_exposure: 11.days.ago, created_at: 20.days.ago }
+    ].each do |workflow_params|
+      dependent_params = {
+        responder: patient,
+        monitoring: false,
+        closed_at: 1.day.ago
+      }.merge(workflow_params)
+      dependent = create(:patient, dependent_params)
+      assert_eligibility(patient, exp_eligibility)
+      dependent.destroy
+    end
+  end
+
+  def monitored_dependent_test(patient, exp_eligibility: true)
+    [
+      { isolation: true },
+      { continuous_exposure: true },
+      { last_date_of_exposure: nil, created_at: 5.days.ago },
+      { last_date_of_exposure: 5.days.ago, created_at: 5.days.ago },
+      { last_date_of_exposure: 11.days.ago, created_at: 20.days.ago }
+    ].each do |workflow_params|
+      dependent = create(
+        :patient,
+        {
+          responder: patient,
+          monitoring: true,
+          closed_at: nil
+        }.merge(workflow_params)
+      )
+      assert_eligibility(patient, exp_eligibility)
+      dependent.destroy
+    end
+  end
+
+  def past_monitoring_period_dependent_test(patient, exp_eligibility: false)
+    [
+      { last_date_of_exposure: nil, created_at: 15.days.ago },
+      { last_date_of_exposure: 15.days.ago, created_at: 15.days.ago },
+      { last_date_of_exposure: 15.days.ago, created_at: 30.days.ago }
+    ].each do |workflow_params|
+      dependent = create(
+        :patient,
+        {
+          responder: patient,
+          monitoring: true,
+          closed_at: nil
+        }.merge(workflow_params)
+      )
+      assert_eligibility(patient, exp_eligibility)
+      dependent.destroy
+    end
+  end
+
+  def ignored_dependent_eligibility_fields_test(patient, exp_eligibility: true)
+    [
+      'E-mailed Web Link', 'SMS Texted Weblink', 'Telephone call',
+      'SMS Text-message', 'Unknown', 'Opt-out', '', nil
+    ].each do |report_method|
+      [
+        { pause_notifications: true },
+        { last_assessment_reminder_sent: 1.hour.ago }
+      ].each do |workflow_params|
+        dependent = create(
+          :patient,
+          {
+            preferred_contact_method: report_method,
+            responder: patient,
+            monitoring: true,
+            closed_at: nil
+          }.merge(workflow_params)
+        )
+        assert_eligibility(patient, exp_eligibility)
+        # Creating an assessment from today SHOULD NOT affect eligibility
+        create(
+          :assessment,
+          patient: dependent,
+          symptomatic: false,
+          created_at: Time.now.getlocal('-04:00').beginning_of_day
+        )
+        assert_eligibility(patient, exp_eligibility)
+        dependent.destroy
+      end
+    end
+  end
+
+  test 'HoH notification eligible because of dependent flows' do
```

Comment:
If we are focused on HoH enelgible but making self eligible with dependent can we simplify this test to focus on a simple obvious HoH is not eligible and go through the cases for dependent here?